marshmallow_recipe/metadata.py

```diff
@@ -27,6 +28,19 @@ def __repr__(self) -> str:
     def __str__(self) -> str:
         return str(self.__values)
 
+    def __hash__(self) -> int:
+        if self.__hash is None:
+            h = 0
```

Comment:
`h` -> `hash` ?

Reply:
hash is a function we use slightly after )))

Reply:
oh, python